config.py

```diff
@@ -6,10 +6,14 @@ class Config(object):
     TESTING = False
     PORT = os.getenv('PORT', 8085)
     LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
+    SECURITY_USER_NAME = os.getenv('SECURITY_USER_NAME')
```

Comment:
These are now separate per service so should be COLLECTION_EXERCISE_USERNAME etc?